authors/apps/authentication/tests/test_login_register.py

```diff
@@ -1,14 +1,25 @@
 """This module tests the login and registration of a user."""
+from django.utils.encoding import force_text
+from django.utils.http import urlsafe_base64_encode, urlsafe_base64_decode
+
 from django.test import TestCase
 
 from rest_framework.test import APIClient
 from rest_framework import status
 
+from authors.apps.authentication.models import User
+
 from authors.apps.authentication.tests.base_test import BaseTest
 
+from authors.apps.authentication.token import account_activation_token
+
 
 class RegistrationAPIViewTestCase(TestCase, BaseTest):
     """Test suite for the api views."""
+    # @staticmethod
```

Comment:
Do we need these lines of code, I can see they are commented out?

Reply:
We, don't. I'm removing them asap